### ISSUE TITLE
aws.ec2.getSubnetIds should return type

### ISSUE THREAD

**Issue by jchannon** [pulumi/pulumi-awsx#358]
When calling `const subnetsids = pulumi.output( aws.ec2.getSubnetIds({vpcId: primaryVpcId,}));` it would be helpful if the `ids` property contained the type of subnet it is eg/public,private,isolated

**Comment by CyrusNajmabadi** [comment]
Right.  Note that you *should* be able to use awsx.ecs.getSubnet to get a value back that you can check the `tags:` of. Pulumi should be creating a tag called `type` there.

We can def look into making this easier to obtain!
